### PR TITLE
Update path to t3skin css for TYPO3 v8

### PR DESCRIPTION
**Pull request by carodb**
Path changed for TYPO3 v8
See also: https://github.com/Ecodev/newsletter/issues/137

### CODE REVIEW

File: Classes/ViewHelpers/Be/ModuleContainerViewHelper.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function render($pageTitle = '')
         $doc = $this->getDocInstance();
         $this->pageRenderer->backPath = '';
         $this->pageRenderer->loadExtJS();
-        $this->pageRenderer->addCssFile('sysext/t3skin/extjs/xtheme-t3skin.css');
+        $this->pageRenderer->addCssFile('sysext/core/Resources/Public/ExtJs/xtheme-t3skin.css');
```
Comment (PowerKiKi) [inline Classes/ViewHelpers/Be/ModuleContainerViewHelper.php]: We'll still have to support TYPO3 7 for a while, so please add condition to support both version similar to, but with correct exact versions where the changed appeared:
 
```php

// From TYPO3 7.4.0 onward we must use EXT prefix
if (version_compare(TYPO3_version, '7.4.0', '>=')) {
    // bla bla
} else {
    // But for TYPO3 6.2 family, we still have to use old style
    // bla bla
}
```

Comment (carodb) [reply Classes/ViewHelpers/Be/ModuleContainerViewHelper.php]: Will do soon!

**Comment by carodb** [comment]
Updated conditional path

**Comment by carodb** [comment]
Can we get this pushed to dev?

**Comment by PowerKiKi** [comment]
Here you go, thanks for the PR !

**Comment by carodb** [comment]
Thanks so much :) 👍

**Pull request merged**